### PR TITLE
Feature: Metadata Drawer

### PR DESCRIPTION
**Pull request by andreashimin**
## Description

- View Metadata Drawer
- Update Metadata Popup
- The caculated field will remain update in model drawer which metadata drawer not provide update caculated fields, only can update it’s metadata.

image

image

image

## Issue ticket number

closes #xx

### CODE REVIEW

File: packages/admin-ui/components/table/MeasureFieldTable.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const { dataSource = [], columns = [] } = props;
+
+  const tableColumns = useMemo(
+    () => [...getMeasureFieldTableColumns(), ...columns],
+    [dataSource]
+  );
+
+  return (
+    <Table
+      rowKey={(record, index) => `${record.fieldName}-${index}`}
```
Comment (fredalai) [inline packages/admin-ui/components/table/MeasureFieldTable.tsx:37]: For the current Ant.d V4 Table element, rowKey will no longer support **index**

Comment (andreashimin) [reply packages/admin-ui/components/table/MeasureFieldTable.tsx:37]: I checked the latest `rc-table`, it supports `index`, though Antd doesn't write it out in their documents. 🤔️
https://github.com/react-component/table/blob/master/src/interface.ts#L118

image

Comment (fredalai) [reply packages/admin-ui/components/table/MeasureFieldTable.tsx:37]: Hmm... I can see the `Warning: [antd: Table] `index` parameter of `rowKey` function is deprecated....` in the console.log. In fact, you will find that index is always 0.

Comment (fredalai) [reply packages/admin-ui/components/table/MeasureFieldTable.tsx:37]: But this is not a big issue and does not affect the core functions. We can also improve it in subsequent PRs.

**Review by fredalai — APPROVED** [review]

File: packages/admin-ui/components/table/MeasureFieldTable.tsx (hunk 1 above)
Comment (andreashimin) [reply packages/admin-ui/components/table/MeasureFieldTable.tsx]: I saw the `Warning` after restarting my process 😢, still don't know why Antd validating it.
I use map to give the key property before table component. [a96a4e0](https://github.com/Canner/vulcan-sql/pull/345/commits/a96a4e07b02643fb3246557afcfc07549f308b68)

**Review by fredalai — APPROVED** [review]
LGTM

**Pull request merged**